src/main/java/app/sami/languageWeb/models/User.java

```diff
@@ -0,0 +1,72 @@
+package app.sami.languageWeb.models;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+
+import java.util.Collection;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+@Entity(name = "users")
+@Table
+@NoArgsConstructor
+@Data
+@Builder
+@AllArgsConstructor
+public class User implements UserDetails{
+    @Id
+    @GeneratedValue
+    private UUID id;
```

Comment:
usually, a user is identified by a `subject` ID, you should keep the `id` in the database as `serial` and add a unique `subject` column that will be used to identify your users.